src/components/tabs/tabs-story-vue.ts

```diff
@@ -1,17 +1,28 @@
 /**
  * @license
  *
- * Copyright IBM Corp. 2019, 2020
+ * Copyright IBM Corp. 2019, 2021
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
 import createVueBindingsFromProps from '../../../.storybook/vue/create-vue-bindings-from-props';
 import { Default as baseDefault } from './tabs-story';
+import styles from './tabs-story.scss';
 
 export { default } from './tabs-story';
 
+const injectStoryStyle = () => {
+  const found = document.querySelector('style#tabs-story');
+  if (!found) {
+    const style = document.createElement('style');
+    style.id = 'tabs-story';
+    style.textContent = styles.cssText;
+    document.head.appendChild(style);
+  }
+};
+
```

Comment:
Looks good thanks @asudoh! Just a small question, why is manual injection necessary for the styles?

Reply:
@kennylam Good question; IIRC Vue template doesn't allow `<style>` tag, and thus above code takes raw DOM approach. In short, we need to create story-specific style.

Reply:
Got it, thanks @asudoh!